docs/self-managed/console-deployment/configuration.md

```diff
@@ -71,7 +80,7 @@ console:
       value: online
 ```
 
-## Using a different OpenID Connect (OIDC) authentication provider than Keycloak
+## Configure an OpenID Connect (OIDC) provider
```

Comment:
I think it is still a good idea to mention keycloak somewhere in the title, just so it is clear that the OIDC provider and keycloak are mutually exclusive. 
Or maybe mention it in a paragraph under the title. 
What do you think @conceptualshark ?